src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+  - /docs/alerts/alert-policies/configuring-alerts/managing-your-alerts 
+freshnessValidatedDate: never
+---
+import alertsSetaQuery from 'images/alerts_screenshot-crop_set-a-query.webp'
+
+import alertsFineTuneAlertSignals from 'images/alerts_screenshot-crop_fine-tune-alert-signals.webp'
+
+import alertsSetaThresholdforanAlertCondition from 'images/alerts_screenshot-crop_set-a-threshold-for-an-alert-condition.webp'
+
+import alertsNameYourAlertCondition from 'images/alerts_screenshot-crop_name-your-alert-condition.webp'
+
+import alertsAlertConditionPage from 'images/alerts_screenshot-crop_alert-condition-page.webp'
+
+New Relic alerts are built with alert conditions. If you want to be notified of any unusual behavior in your stack, you'll first need to create an alert condition. An alert condition is essentially a container that you create to define what conditions need to be met before you're notified of any unusual behavior. 
+
+<img
+    title="alert condition homepage"
+    alt="A screenshot showing the alert condition homepage"
+    src={alertsAlertConditionPage}
+/>
+<figcaption>
+To use New Relic's universal alert condition experience, go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > **Alerts & AI**, select **Alert Conditions** in the left navigation.
+</figcaption>
+
+With our **Alert Conditions** page you can:
+* Browse all of your alert conditions 
+* Easily identify any open issues
+* Open existing alert conditions to edit thresholds or signal behavior 
+* Create a new alert condition 
+
+This doc will walk you through how to edit and change [an _existing_ alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#edit-existing-alert-condition), and creating [new_ alert condition_](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#create-alert-condition]). 
+
+## Create a new alert condition [#create-alert-condition]
+
+For this example, we are going to show you how to manually create a new alert condition using the **Alert Condition** UI. But, if you're just beginning to use New Relic, we suggest creating an alert condition from a chart. To get started with alerts, review our [tutorial about creating your first alert](/docs/tutorial-create-alerts/create-an-alert/). 
+
+<Steps>
+<Step>
+An alert condition is essentially a container that you create to define what conditions need to be met before you're notified of any unusual behavior.
+
+<img
+    title="set a query"
+    alt="A screenshot showing a user how to set a query for an alert condition"
+    src={alertsSetaQuery}
+/>
+
+<figcaption>
+To create a new alert, go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > **Alerts & AI**, select **Alert Conditions** in the left navigation, then select **New alert condition**.
+</figcaption>
+
+<CollapserGroup>
```

Comment:
I almost looked past this in the doc. This can be integrated into your first step, which can be titled 'Set your initial query' or even 'Define your initial query'